app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [OPTIONAL] I think here you don't need to add again `foreign_key` as it has already been added in the user.rb file 🙃 🙃